From e3cc261f648e2ee9641224e9260945c63d9328c9 Mon Sep 17 00:00:00 2001
From: Loc Pham <loc.pham.xh@rvc.renesas.com>
Date: Mon, 27 Nov 2017 14:54:19 +0700
Subject: [PATCH 327/433] drm: panel-simple: Correct resolution of the EDT
 ETM043080DH6-GP panel

ETM043080DH6-GP is 480x272 TFT Display with capacitive touchscreen.
This patch changes panel resolution from 480x288 to 480x272 to match HW
specification.

Signed-off-by: Loc Pham <loc.pham.xh@rvc.renesas.com>
Signed-off-by: Hao Bui <hao.bui.yg@renesas.com>
---
 drivers/gpu/drm/panel/panel-simple.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/gpu/drm/panel/panel-simple.c b/drivers/gpu/drm/panel/panel-simple.c
index 52e5c50..5fc121c 100644
--- a/drivers/gpu/drm/panel/panel-simple.c
+++ b/drivers/gpu/drm/panel/panel-simple.c
@@ -626,10 +626,10 @@ static const struct drm_display_mode edt_etm043080dh6gp_mode = {
 	 * fb_align
 	 */
 
-	.vdisplay = 288,
-	.vsync_start = 288 + 2,
-	.vsync_end = 288 + 2 + 4,
-	.vtotal = 288 + 2 + 4 + 10,
+	.vdisplay = 272,
+	.vsync_start = 272 + 2,
+	.vsync_end = 272 + 2 + 4,
+	.vtotal = 272 + 2 + 4 + 10,
 	.vrefresh = 60,
 };
 
-- 
2.7.4

